WebMvc/WebMvc.csproj

```
@@ -11,7 +11,7 @@
     <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.7.9" />
     <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.9" />
     <PackageReference Include="Polly" Version="7.1.0" />
-    <PackageReference Include="Stripe.net" Version="27.19.0" />
+    <PackageReference Include="Stripe.net" Version="17.8.0" />
```

```suggestion
    <PackageReference Include="Stripe.net" Version="27.19.0" />
```
I used the latest updated one because for the OrderApi integration in WebMVC, I changed all the methods so we can use the latest version instead of downgrading to an older version. If it doesn't work, we can then downgrade it, but currently the master branch OrderAPI integration uses methods and properties suitable for the newest version of Stripe.

---

If we upgrade the version, it will show a red underline some parts. We can just leave it as the upgraded version for now and downgrade it if we need to later!